specs/login/cluster-init/23-pyxis.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Is this file needed?   It's identical to the script from 'default' (and it's not in the "scripts" directory)

Reply:
yes the default is used only for execute, I will reorganize the whole project